fvtest/tril/README.md

```diff
@@ -45,54 +45,31 @@ on your machine
 
 2. Clone the Eclipse OMR repo:
 
-```sh
-git clone https://github.com/eclipse/omr.git
-```
-
-3. Build JitBuilder
-
-```sh
-cd omr
-./configure SPEC=x86-64 --enable-OMR_JITBUILDER # set SPEC for your platfrom
-cd jitbuilder
-make -j4
-cd ..
-```
-
-4. Build the Tril library
-
-```sh
-cd fvtest/tril
-mkdir build
-cd build
-cmake -GNinja ..
-cmake --build .
-```
+    ```sh
+    git clone https://github.com/eclipse/omr.git
+    ```
 
-5. You should now have a static archive called `libtril.a`
-
-### Building the Mandelbrot example
+3. Build tril using `cmake`. To ensure OMR gets built in a compatible
+   configuration, we currently reccomend you use the CMake cache that drives
+   our TravisCI builds.
 
-Now that the Tril library was built, you should be able to build any of the
-provided examples.
+    ```sh
+    mkdir build
+    cd build
+    cmake -GNinja -C../cmake/caches/Travis.cmake ..
+    ```
 
-1. `cd` into the Tril top level directory (`cd ../..` if continuing from
-   previous section)
+    At this point you can use whatever generated build system to build
+    tril: 
 
-2. Go to the Mandelbrot example directory and build
+    ```
+    make tril
```

Comment:
The earlier command shows using the ninja generator, so this should use ninja too.